README.md

```diff
@@ -115,35 +123,6 @@ The script will be executed as `./search --json <json>` where the JSON string ha
 }
 ```
 
-# Docker Container Contract
```

Comment:
Where did this go? Why are we removing it?

Reply:
This is a duplicate section that slipped in somewhere from a bad copy-paste... the updated section is still there.